models/op_token/op_token_optimism_initial_allocations.sql

```diff
@@ -0,0 +1,40 @@
+
+
+{{ config(
+        schema = 'op_token_optimism',
+        alias='initial_allocations'
+        , unique_key = ['allocation_category','allocation_subcategory']
+        , post_hook='{{ expose_spells(\'["optimism"]\',
+                                  "project",
+                                  "ovm_optimism",
+                                  \'["msilb7"]\') }}'
+  )
+}}
+
+, initial_allocation_percentages AS (
```

Comment:
this is the reason why the slim ci is failing. seems you are missing a CTE?

Reply:
copy and paste error. resolved